mne/viz/_3d.py

```diff
@@ -473,36 +492,195 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
     -------
     fig : instance of mlab.Figure
         The mayavi figure.
+
+    See Also
+    --------
+    mne.viz.plot_bem
     """
-    from ..forward import _create_meg_coils
-    mlab = _import_mlab()
+    if source is not None and bem is not None:
+        raise ValueError('Only one of "source" and "bem" must be provided, '
+                         'got %s and %s respectively.' % (source, bem))
+    surfaces = []
+    if skull:
+        if skull is True or 'outer_skull' in skull:
+            surfaces.append('outer_skull')
+        if 'inner_skull' in skull:
+            surfaces.append('inner_skull')
+
+    if head:
+        surfaces.append('outer_skin')
+    elif head is None:
+        surfaces.append('ecog_head')  # special case
+
+    if brain is True:
+        surfaces.append('pial')
+    elif brain is None:
+        surfaces.append('ecog_brain')  # special case
+    elif isinstance(brain, str):
+        surfaces.append(brain)
+
     if meg_sensors is False:  # old behavior
-        meg_sensors = 'helmet'
+        meg = ['helmet']
     elif meg_sensors is True:
-        meg_sensors = ['helmet', 'sensors']
-    if eeg_sensors is False:
-        eeg_sensors = []
-    elif eeg_sensors is True:
-        eeg_sensors = 'original'
-    if isinstance(eeg_sensors, string_types):
-        eeg_sensors = [eeg_sensors]
-    if isinstance(meg_sensors, string_types):
-        meg_sensors = [meg_sensors]
-    for kind, var in zip(('eeg', 'meg'), (eeg_sensors, meg_sensors)):
+        meg = ['helmet', 'sensors']
+    elif isinstance(meg_sensors, str):
+        meg = [meg_sensors]
+    else:
+        meg = []
+    if ref_meg:
+        meg.append('ref')
+
+    return plot_alignment(info, trans=trans, subject=subject,
+                          subjects_dir=subjects_dir, surfaces=surfaces,
+                          coord_frame=coord_frame, meg=meg, eeg=eeg_sensors,
+                          dig=dig, ecog=ecog_sensors, src=src,
+                          mri_fiducials=mri_fiducials, bem=bem,
+                          verbose=verbose)
+
+
+@verbose
+def plot_alignment(info, trans='auto', subject=None, subjects_dir=None,
+                   surfaces=('head',), coord_frame='head',
+                   meg=('helmet', 'sensors'), eeg='original',
+                   dig=False, ecog=True, src=None, mri_fiducials=False,
+                   bem=None, verbose=None):
+    """Plot head, sensor, and source space alignment in 3D.
+
+    This function serves the purpose of checking the validity of the many
+    different steps of source reconstruction:
+
+    - Transform matrix (keywords ``trans``, ``meg`` and ``mri_fiducials``),
+    - BEM surfaces (keywords ``bem`` and ``surfaces``),
+    - sphere conductor model (keywords ``bem`` and ``surfaces``) and
+    - source space (keywords ``surfaces`` and ``src``).
+
+    Parameters
+    ----------
+    info : dict
+        The measurement info.
+    trans : str | 'auto' | dict | None
```

Comment:
I actually never liked the `auto`, it is too magical. Let's make this a required argument by not giving a default.